pom.xml

```diff
@@ -148,58 +103,18 @@
       <artifactId>spring-expression</artifactId>
       <version>${spring.version}</version>
     </dependency>
+    
     <dependency>
-      <groupId>org.springframework</groupId>
-      <artifactId>spring-jdbc</artifactId>
-      <version>${spring.version}</version>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.testng</groupId>
-      <artifactId>testng</artifactId>
-      <version>7.0.0-beta1</version>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.mockito</groupId>
-      <artifactId>mockito-all</artifactId>
-      <version>2.0.2-beta</version>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>net.sf.jacob-project</groupId>
-      <artifactId>jacob</artifactId>
-      <version>${jacob.version}</version>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>net.sf.jacob-project</groupId>
-      <artifactId>jacob</artifactId>
-      <version>${jacob.version}</version>
-      <classifier>x86</classifier>
-      <type>dll</type>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>net.sf.jacob-project</groupId>
-      <artifactId>jacob</artifactId>
-      <version>${jacob.version}</version>
-      <classifier>x64</classifier>
-      <type>dll</type>
-      <scope>test</scope>
-    </dependency>
-
-    <dependency>
-      <groupId>quartz</groupId>
+      <groupId>org.quartz-scheduler</groupId>
```

Comment:
I didn't know Quartz scheduler was used in DL, but okay!

Reply:
I have verified by running unit tests and integration tests on Mac and manual tests on Mac and Windows.